src/Ifelse.java

```diff
@@ -0,0 +1,199 @@
+import java.util.Scanner;
+
+public class Ifelse {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        // Opdracht 1
+        int number1 = 3;
+        opdracht1(number1);
+
+        // Opdracht 2
+        float cijfer = 8f;
+        opdracht2(cijfer);
+
+        // Opdracht 3
+        int count1 = 1;
+        int count2 = 50;
+        opdracht3(count1, count2);
+
+        // Opdracht 4
+        int number2 = 1;
+        opdracht4(number2);
+
+        // Opdracht 5
+        int leeftijd = 80;
+        opdracht5(leeftijd);
+
+        // Opdracht 6
+        String text = "Dit is meer dan 10 woorden";
+        opdracht6(text);
+
+        // Opdracht 7
+        int jaar = 5;
+        opdracht7(jaar);
+
+        // Opdracht 8
+        System.out.println("Voer het eerste getal in: ");
+        int getal1 = scanner.nextInt();
+        System.out.println("Voer het tweede getal in: ");
+        int getal2 = scanner.nextInt();
+        System.out.println("Voer het derde getal in: ");
+        int getal3 = scanner.nextInt();
+
+        opdracht8(getal1, getal2, getal3);
+
+        // Opdracht 9
+        System.out.println("Voer het eerste getal in: ");
+        int driehoek1 = scanner.nextInt();
+        System.out.println("Voer het tweede getal in: ");
+        int driehoek2 = scanner.nextInt();
+        int driehoek3 = 180 - (driehoek1 + driehoek2);
+
+        opdracht9(driehoek1, driehoek2, driehoek3);
+
+        // Opdracht 10
+        System.out.println("Voer het eerste getal in: ");
+        int count3 = scanner.nextInt();
+        System.out.println("Voer het tweede getal in: ");
+        int count4 = scanner.nextInt();
+        System.out.println("Voeg een operator in ( + - * / )");
+        String operator = scanner.next();
+
+        opdracht10(count3, count4, operator);
+    }
+
+    // Opdracht 1
+    private static void opdracht1(int number1) {
+        if (number1 % 2 != 0) {
+            System.out.println("Het getal " + number1 + " is oneven");
+        } else {
+            System.out.println("Het getal "+ number1 + " is even.");
+        }
+    }
+
+    // Opdracht 2
+    private static void opdracht2(float cijfer) {
```

Comment:
Je hebt zelfs de plusjes en de minnetjes er bij gepakt. Daarom krijg jij een A for effort! Maar wat is er met de D en de E gebeurt?